packages/manager/modules/pci/src/projects/project/translations/Messages_fr_FR.json

```
@@ -19,5 +19,6 @@
   "pci_projects_project_welcome": "Bienvenue sur la nouvelle interface Public Cloud",
   "pci_projects_project_keystone_upgrade": "Changement important le 23 juin 2020, merci d'anticiper dès maintenant le passage à <strong>Keystone v3</strong>.",
   "pci_projects_project_keystone_upgrade_more": "En savoir plus",
-  "pci_projects_project_quota_warning_message": "Par mesure de sécurité, votre projet est soumis à un quota. Si vous voulez augmenter ce quota, rendez-vous sur la page <a data-ui-sref=\"{{ quotaUrl }}\">Quota and localization</a>."
+  "pci_projects_project_quota_warning_message": "Par mesure de sécurité, votre projet est soumis à un quota. Si vous voulez augmenter ce quota, rendez-vous sur la page <a data-ui-sref=\"{{ quotaUrl }}\">Quota and localization</a>.",
+  "pci_projects_project_quota_threshold_warning_message": "Vos niveaux de quotas arrivent à leur maximum, <a data-ui-sref=\"{{ quotaUrl }}\">cliquez ici pour gérer vos quota</a>."
```

```suggestion
  "pci_projects_project_quota_threshold_warning_message": "Vos niveaux de quotas arrivent à leur maximum, <a href=\"{{ quotaUrl }}\">cliquez ici pour gérer vos quota</a>."
```
we should avoid `ui-sref` in translations (even though it has been done above)